### ISSUE TITLE
Project & WSDL Naming Convention

### ISSUE THREAD

**Issue by swadi** [jacobono/gradle-wsdl-plugin#12]
What is the reason behind restricting the naming convention to end with a -ws? and what is the need for ending a WSDL name with "Service"? Wouldn't this hinder usage of this plugin in existing projects? Or is there a workaround?

**Comment by jszuslik** [comment]
Any updates on this? I have the same issue and need this.

**Comment by pvdissel** [comment]
yeah, no fun to have plugins enforce project and wsdl file namings
